README.md

```diff
@@ -302,7 +302,13 @@ If you need to display utf encoded characters, add this to your pdf views or lay
 ```html
 <meta charset="utf-8" />
 ```
-
+If you need to return a PDF in a controller with Rails in API mode:
+```ruby
+# teamplate in `app/views/controller_name/action_name.html.erb` and layout in `app/views/layouts/pdf.html.erb`
```

Comment:
Typo here: teamplate => template, and I think this comment isn't necessary.

Reply:
Ok thanks